src/Serilog.Sinks.Elasticsearch/project.json

```diff
@@ -0,0 +1,61 @@
+{
+    "version": "4.1.0",
```

Comment:
For the new build setup to work this needs to be `4.1.0-*` - the scripts will set the suffix correctly to nothing (the `-` disappears) or `-branch-nnnnnn` depending on whether or not `master` is being built.